Remotion/Web/Development.WebTesting/WebDriver/Configuration/InternetExplorer/InternetExplorerConfiguration.cs

```diff
@@ -63,7 +56,7 @@ public override IBrowserFactory BrowserFactory
 
     public override IDownloadHelper DownloadHelper
     {
-      get { return _downloadHelper; }
+      get { throw new NotSupportedException ("RM-7457: Support for Internet Explorer in web tests has been removed."); }

```

Comment:
I'm not sure yet how we should handle this in 1.20. Adding a `NullDownloadHelper` that always succeeds would be non-breaking but cause false negatives.

Is this one of those instances where a breaking change would be alright? Since it's cordinated?

Reply:
yes. in both 1.20 and trunk, the InternetExplorer APIs should have an Obsolete-Attribute anyway.

Reply:
Oh, does that mean I should move `InternetExplorerConfiguration` and everything associated with it into our test projects?

Reply:
Correct (if possible)